docs/source/tutorial/interfaces.rst

```diff
@@ -193,16 +192,15 @@ the type of the ``show`` method:
      Main> :t show
      Prelude.show : {0 a : Type} -> Show a => a -> String
 
-However some use cases require that some of the parameters are available at runtime.
+However, some use cases require that some of the parameters are available at runtime.
 We may for instance want to declare an interface for ``Storable`` types. The constraint
 ``Storable a size`` means that we can store values of type ``a`` in a ``Buffer`` in
 exactly ``size`` bytes.
 
 If the user provides a method to read a value for such a type ``a`` at a given offset,
 then we can read the ``k`` th element stored in the buffer by computing the appropriate
 offset from ``k`` and ``size``. This is demonstrated by providing a default implementation
-for the method ``peekElementOff`` implemented in terms of ``peekByteOff`` and the parameter
-``size``.
+for the method ``peekElementOff`` in terms of ``peekByteOff`` and the parameter ``size``.
```

Comment:
A comma after \`\`peekElementOff\`\` might more clearly convey the intended phrasing?